docs/source/developers/java/building.rst

```diff
@@ -227,6 +223,7 @@ CMake
           -DCMAKE_INSTALL_PREFIX=java-dist \
           -DCMAKE_UNITY_BUILD=ON
       $ cmake --build cpp-jni --target install --config Release
+      $ export JAVA_JNI_CMAKE_ARGS="-DProtobuf_ROOT=$PWD/cpp-jni/protobuf_ep-install"
```

Comment:
We need to specify this in the following `cmake ...` command line because `JAVA_JNI_CMAKE_ARGS` is only for `ci/scripts/java_*_build.sh`.